src/test/ui/generic-associated-types/issue-91139.rs

```diff
@@ -1,3 +1,5 @@
+//~ ERROR the parameter type `T` may not live long enough
```

Comment:
This message is emitted with `DUMMY_SP`. This is unfortunate. Any idea where it comes from?

Reply:
yeah, I forgot to comment on this one. This indeed comes from https://github.com/rust-lang/rust/blob/37928f59865b5ff53b95fe2f3c02491890a4129d/compiler/rustc_borrowck/src/type_check/free_region_relations.rs#L305
which I've fixed in #99818 to make it point to the closure body instead.